src/sentry/static/sentry/app/components/issueDiff.jsx

```diff
@@ -0,0 +1,102 @@
+import React, {PropTypes} from 'react';
+import classNames from 'classnames';
+
+import ApiMixin from '../mixins/apiMixin';
+import LoadingIndicator from './loadingIndicator';
+import rawStacktraceContent from './events/interfaces/rawStacktraceContent';
+
+import '../../less/components/issueDiff.less';
+const IssueDiff = React.createClass({
+  propTypes: {
+    baseIssueId: PropTypes.string.isRequired,
+    targetIssueId: PropTypes.string.isRequired,
+    baseEventId: PropTypes.string.isRequired,
+    targetEventId: PropTypes.string.isRequired
+  },
+
+  mixins: [ApiMixin],
+
+  getDefaultProps() {
+    return {
+      baseEventId: 'latest',
+      targetEventId: 'latest'
+    };
+  },
+
+  getInitialState() {
+    return {
+      loading: true,
+      baseEvent: {},
+      targetEvent: {},
+      Diff: null
```

Comment:
Maybe a comment like, "`Diff` component code is loaded asynchronously, this value is set when that code becomes available"